src/services/email/email.definition.ts

```diff
@@ -0,0 +1,9 @@
+export interface EmailInfo {
+  to: Array<string>;
```

Comment:
only one of cc, bcc, to is required, not necessary that to is always filled in (unless sendgrid requires it to be)

Reply:
Yeah that's true. To be frank cc, bcc and to are all optional properties of SendGridMail's type

Reply:
I'll make 'to' optional as well